### PR TITLE
Use es classes

### PR DESCRIPTION
**Pull request by JustinBeckwith**
BREAKING CHANGE: This library is now compatible with es module import syntax.  

#### Old Code
```js
const logging = require('@google-cloud/logging')();
// or...
const Logging = require('@google-cloud/logging');
const logging = new Logging();
```

#### New Code
```js
const {Logging} = require('@google-cloud/logging');
const logging = new Logging();
```

### CODE REVIEW

**Comment by stephenplusplus** [comment]
Tiiiiny coverage loss :\

File: src/index.js [hunk 1]
```diff
@@ -109,853 +107,773 @@ const Sink = require('./sink.js');
...
-    v2.MetricsServiceV2Client,
-  ];
-  for (let clientClass of clientClasses) {
-    for (let scope of clientClass.scopes) {
-      if (clientClasses.indexOf(scope) === -1) {
-        scopes.push(scope);
+class Logging {
+  constructor(options) {
+    if (!(this instanceof Logging)) {
+      return new Logging(options);
```
Comment (stephenplusplus) [inline src/index.js]: I think this is causing the coverage loss: https://codecov.io/gh/googleapis/nodejs-logging/pull/219/diff?src=pr&el=tree#D8-113

Do we still need it?

Comment (JustinBeckwith) [reply src/index.js]: Great call.  Removed!

**Comment by JustinBeckwith** [comment]
*gentle nudge* @stephenplusplus

**Pull request merged**